### ISSUE TITLE
Opening a localized component always open the same tab

### ISSUE THREAD

**Issue by pascalrobert** [wocommunity/wolips#43]
If you use localized component (Main.wo in Chinese.lproj, Main.wo in English.lproj, etc.), no matter which component you are opening, the tab in Component Editor is always the same. For example, the HelloWorld example from Apple have components in Chinese, French, German, Japanese and Spanish. If I double-click on Main.wo in Chinese.lproj, it's the Spanish tab that opens. Same thing if I open the component in French.

In summary, when you open a localized component, the tab in Component Editor should be the selected component.
